exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -471,6 +428,66 @@ func (b *Bitfinex) wsHandleData(respRaw []byte) error {
 	return nil
 }
 
+func (b *Bitfinex) handleWSEvent(respRaw []byte) error {
+	event, err := jsonparser.GetUnsafeString(respRaw, "event")
```

Comment:
GetUnsafeString means it's not safe to return the var or store a pointer to it.
We're using it within that context.

Considered calling it eventUnsafe to avoid future mis-uses